src/Uno.UI/UI/Xaml/Documents/Inline.skia.cs

```diff
@@ -115,17 +118,18 @@ protected override void OnFontWeightChanged()
 			else
 			{
 				skTypeFace = SKTypeface.FromFamilyName(name, skWeight, skWidth, skSlant);
+			}
 
+			if (skTypeFace == null)

```

Comment:
I don't understand why this is moved outside the `else`?

The code comment says this can be null when `FromFamilyName` is called, so this only makes sense in the `else` block.

I also don't understand where the behavior change comes from?

Reply:
If this is reachable because `SKTypeface.FromFile(filePath)` can be null, it would be good to update the below comment so that it's clear.